### PR TITLE
docs: add notes for Windows users

### PR DESCRIPTION
**Pull request by jennings**
I discovered there's at least one other jj user on Windows after [a question came up in chat](https://discord.com/channels/968932220549103686/1200419885008830666) about dealing with `@` being part of PowerShell's syntax. I thought it would be a good idea to list out the things Windows users probably need to be aware of.

# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [X] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (cli/src/config-schema.json)
- [ ] I have added tests to cover my changes

### CODE REVIEW

File: docs/windows.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Working on Windows
+
+jj works the same on all platforms, but there are some caveats that Windows
```
Comment (PhilipMetzger) [inline docs/windows.md:3]: Nit: Use `jj` or Jujutsu throughout the whole doc. Atm it is not consistent with the rest.

File: docs/windows.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## Pagination
+
+[Pagination is disabled by default on
+Windows](https://github.com/martinvonz/jj/issues/2040) because Windows doesn't
+ship with a usable pager.
+
+If you have Git installed, you can use Git's pager and re-enable pagination:
+
+```
```
Comment (PhilipMetzger) [inline docs/windows.md:21]: Nit: Missing language for `mkdocs`, I assume it's Powershell.

File: docs/windows.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+jj config set --user ui.pager '["C:\\Program Files\\Git\\usr\\bin\\less.exe", "-FRX"]'
+jj config set --user ui.paginate auto
+```
+
+## Typing "@" in PowerShell
+
+PowerShell uses `@` as part the [array sub-expression operator][array-op], so it
+often needs to be escaped or quoted in commands:
+
+```
```
Comment (PhilipMetzger) [inline docs/windows.md:31]: Ditto

File: docs/windows.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+jj log -r `@
+jj log -r '@'
+```
+
+One solution is to create a revset alias. For example, to make HEAD an alias for
+@:
```
Comment (PhilipMetzger) [inline docs/windows.md:37]: Nit: should also be `@`.

File: mkdocs.yml
Comment (PhilipMetzger) [inline mkdocs.yml:1]: Can you remove the unrelated changes?

File: docs/windows.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+jj does not honor `.gitattributes` and does not have a setting like git's
+`core.autocrlf`. This means that line endings will be checked out exactly as
+they are committed and committed exactly as authored. This is true on all
+platforms, but Windows users are most likely to miss CRLF conversion.
+
+## Pagination
+
+[Pagination is disabled by default on
+Windows](https://github.com/martinvonz/jj/issues/2040) because Windows doesn't
```
Comment (PhilipMetzger) [inline docs/windows.md:16]: Optional: Also use a named link here and move it to EOF.

File: docs/windows.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+One solution is to create a revset alias. For example, to make HEAD an alias for
+@:
+
+```
+jj config set --user revset-aliases.HEAD '@'
+jj log -r HEAD
+```
+
+[array-op]: https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_arrays?view=powershell-7.4#the-array-sub-expression-operator
```
Comment (PhilipMetzger) [inline docs/windows.md:44]: Optional: Move to EOF.

File: docs/windows.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
+# Working on Windows
+
+jj works the same on all platforms, but there are some caveats that Windows
+users should be aware of.
+
+## Line endings are not converted
+
+jj does not honor `.gitattributes` and does not have a setting like git's
```
Comment (PhilipMetzger) [inline docs/windows.md:8]: Nit: "git's" => "Git's"

**Review by PhilipMetzger — CHANGES_REQUESTED** [review]
Some small nits. But after that it should be good to land.

File: docs/windows.md (hunk 1 above)
Comment (jennings) [reply docs/windows.md]: Updated to "Jujutsu" everywhere.

File: docs/windows.md (hunk 2 above)
Comment (jennings) [reply docs/windows.md]: Added the `powershell` language tag, and added the PowerShell prompt `PS>` to all commands, since the shell examples show the `$` prompt.

File: docs/windows.md (hunk 3 above)
Comment (jennings) [reply docs/windows.md]: Done.

File: mkdocs.yml
Comment (jennings) [reply mkdocs.yml:1]: Ugh, sorry, I didn't realize I had "format on save" set. Fixed.

File: docs/windows.md (hunk 6 above)
Comment (jennings) [reply docs/windows.md]: Done.

File: docs/windows.md (hunk 5 above)
Comment (jennings) [reply docs/windows.md]: Done.

File: docs/windows.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+PS> jj config set --user ui.paginate auto
+```
+
+## Typing `@` in PowerShell
+
+PowerShell uses `@` as part the [array sub-expression operator][array-op], so it
+often needs to be escaped or quoted in commands:
+
+```powershell
+PS> jj log -r `@
```
Comment (PhilipMetzger) [inline docs/windows.md:31]: Just a question, this singular \` works for Windows? At work I've been mostly using `jj log -r "@"`.

**Review by PhilipMetzger — APPROVED** [review]
LGTM

File: docs/windows.md (hunk 8 above)
Comment (jennings) [reply docs/windows.md:31]: Yes, backtick is the escape character in PowerShell (newline is ``` `n ```, etc.):

```
> jj log -r `@
@  qyqrpwvz stephen.g.jennings@gmail.com 16 minutes ago windows-docs default@ 1a3b8731
│  docs: add notes for Windows users
~
```

Comment (PhilipMetzger) [reply docs/windows.md:31]: TIL, thanks for clarifying.
